React/tic-tac/src/services/UserService.js

```diff
@@ -0,0 +1,21 @@
+import UserService from '@config/api';
+import LogInTrue from '@redux/auth/actions';
+
+export function LogIn(payload) {
+  return dispatch => {
```

Comment:
Try not to mix redux with services. Services should handle only the API requests. The response is what you should be handling within the action definition.